Formula/multitime.rb

```diff
@@ -0,0 +1,24 @@
+class Multitime < Formula
+  desc "Time command execution over multiple executions"
+  homepage "https://tratt.net/laurie/src/multitime/"
+  url "https://github.com/ltratt/multitime/archive/multitime-1.4.tar.gz"
+  sha256 "31597066239896ee74a3aaaea3b22931a50a1ec1470090c5457ef35500c44249"
+  license "MIT"
+
+  depends_on "autoconf" => :build
+
+  def install
+    system "autoconf"
+    system "autoheader"
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--mandir=#{man}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/multitime", "-n", "2", "sleep", "1"
```

Comment:
Can we do an `assert_match` or, better, `assert_equal` to check the output here?

Reply:
I don't think that would work very well as the output can very a little bit on the last decimal position.
e.g:
```
===> multitime results
1: sleep 1
            Mean        Std.Dev.    Min         Median      Max
real        1.004       0.001       1.003       1.004       1.006
user        0.001       0.000       0.000       0.001       0.001
sys         0.002       0.001       0.001       0.002       0.003
```

I could check that the first line is present, but that doesn't seem super useful.

Reply:
`assert_match` can take regex arguments.

Reply:
How about this:
```
output = shell_output("#{bin}/multitime -n 2 sleep 1 2>&1")
assert_match(/((real|user|sys)\s+(\d\.\d\d\d\s*)+){3}/m, output)
```

(I force pushed this change to my branch.)